src/lambda/commands/deploySamApplication.ts

```diff
@@ -324,3 +330,36 @@ function getDefaultWindowFunctions(): WindowFunctions {
         showInformationMessage: vscode.window.showInformationMessage,
     }
 }
+
+function notifyDuringLongDeployment(deployParameters: DeploySamApplicationParameters): NodeJS.Timeout {
+    return setTimeout(async () => {
```

Comment:
Normally we should at least exercise this code with a basic test. But we can skip that for now, because we will need to revisit this for "sam sync" anyway.